docs/monoDrive_home/Viewport.md

```diff
@@ -1,49 +1,36 @@
 # Viewport
 
-## Viewport
-The monoDrive Viewport sensor allows you to configure the information displayed on your screen.
+The Viewport sensor allows configuration of the camera that is displayed in the 
+monoDrive Simulator or Scenario Editor. 
+
```

Comment:
The Viewport sensor **is a** configuration of the camera that is displayed in the monoDrive Simulator or Scenario Editor.

Reply:
Fixed. Changed to:

```
The Viewport sensor allows a client to configure the camera that is displayed in 
the monoDrive Simulator or Scenario Editor.
```